idemp-rest-test/testing/src/models/valueObject.ts

```diff
@@ -73,7 +77,7 @@ export class ValueObject {
 	// int m_processRecord_ID = 0;
 
 	sessionToken?: string;
-	isSales = true;
+	isSalesTransaction = true;
```

Comment:
I was curious how Chuck dealt with sales transactions in his value object and he does so by reading from the document type. Can we do the same thing and then remove this property?

Reply:
![document_type_snap](https://user-images.githubusercontent.com/6028400/187862150-e44a333a-f6ff-4426-8d61-8645585e20ab.png)

The document type is defined as a string. Should we change the type and update the dependencies, if any? Thanks

Reply:
Hey Kevin, at this stage I'd appreciate if you'd write the tests. I'll be out most of next week and I don't think this issue should drag into the fourth sprint. Thanks

Reply:
Sure thing. Done!